azure-pipelines.yml

```diff
@@ -119,6 +119,19 @@ jobs:
     # Only run on pull requests to check new commits only
     condition: eq(variables['Build.Reason'], 'PullRequest')
     displayName: Check commit metadata
+  - bash: |
+      find . -name '*.vendor.hjson' \
+        | grep -v /vendor/lowrisc_ibex/vendor/ \
+        | xargs -n1 util/vendor.py --check-patches --verbose \
+        && git diff --exit-code
+      if [[ $? != 0 ]]; then
+        echo -n "##vso[task.logissue type=error]"
+        echo "Vendored repositories not up-to-date. Re-clone them with util/vendor.py."
```

Comment:
```suggestion
        echo "Vendored repositories not up-to-date. Update them with util/vendor.py."
```

Reply:
Ah, sorry, i chose this original wording because i didn't want users to use `vendor.py --update`, so I was avoiding using the word "update".

Reply:
Ah ok, fine either way. Maybe "Run util/vendor.py to fix."